pkg/kubectl/cmd/get.go

```diff
@@ -182,13 +183,22 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 		return cmdutil.UsageError(cmd, usageString)
 	}
 
-	// always show resources when getting by name or filename
+	// determine if args contains "all"
+	for _, a := range args {
+		if a == "all" {
```

Comment:
This will match situations where resources are named 'all' 

`kubectl get pods all`

I'd either address this in a follow up for a quicker approval and submission, or fix it by matching the parsing logic for the 'all' keyword